README.md

```diff
@@ -227,6 +227,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Poloniex](https://poloniex.com/support/api/) | US based digital asset exchange | `apiKey` | Yes | Unknown |
 | [Technical Analysis](https://technical-analysis-api.com) | Cryptocurrency prices and technical analysis | `apiKey` | Yes | No |
 | [VALR](https://docs.valr.com/) | Cryptocurrency Exchange based in South Africa | `apiKey` | Yes | Unknown |
+| [WallstreetBets Api](https://dashboard.nbshare.io/apps/reddit/api/ | WallstreetBets Stock Comments Sentiment Analysis | No | Yes | Unknown |
```

Comment:
```suggestion
| [WallstreetBets](https://dashboard.nbshare.io/apps/reddit/api/) | WallstreetBets Stock Comments Sentiment Analysis | No | Yes | Unknown |
```

Remove `API` of title and close link with `)` character, please.

Reply:
my bad, thanks for correcting it. I have accepted the suggestions.